packages/lesswrong/components/ea-forum/giving-portal/ElectionCandidate.tsx

```diff
@@ -91,10 +100,15 @@ const styles = (theme: ThemeType) => ({
   },
 });
 
-const ElectionCandidate = ({candidate, classes}: {
+const ElectionCandidate = ({candidate, type="preVote", selected, onSelect, classes}: {
   candidate: ElectionCandidateBasicInfo,
+  type?: "preVote" | "select",
```

Comment:
I made this component do double duty as a pre-vote version and a "select" version. If we needed to maintain this for longer I think this wouldn't be ideal, but it saves a bit of boilerplate and we won't need to make any changes to it after december